.github/workflows/release.yml

```diff
@@ -50,5 +50,6 @@ jobs:
     - name: Create pull-request for release changes
       uses: peter-evans/create-pull-request@v3
```

Comment:
Does this allows to specify the files to be committed? if yes, I would be explicit and specify them